aws/v19.1.0/release.yaml

```
@@ -0,0 +1,169 @@
+    version: 1.24.1-gs9
+  - componentVersion: 1.21.0
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: aws-ebs-csi-driver
+    version: 2.25.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: cert-exporter
+    version: 2.6.0
+  - componentVersion: 1.12.3
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: cert-manager
+    version: 3.3.0
+  - name: chart-operator
+    version: 2.35.0
+  - componentVersion: 1.13.6
+    dependsOn:
+    - aws-cloud-controller-manager
+    name: cilium
+    version: 0.12.0
+  - componentVersion: 1.24.3
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: cluster-autoscaler
+    version: 1.24.3
+  - componentVersion: 1.9.3
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    name: coredns
+    version: 1.18.1
+  - componentVersion: 0.11.0
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: external-dns
+    version: 2.39.0
+  - componentVersion: 0.6.1
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: metrics-server
+    version: 2.2.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: net-exporter
+    version: 1.17.0
+  - componentVersion: 1.3.1
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: node-exporter
+    version: 1.17.1
+  - componentVersion: 0.14.0
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: vertical-pod-autoscaler
+    version: 4.0.0
+  - name: vertical-pod-autoscaler-crd
+    version: 2.0.1
+  - dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: etcd-kubernetes-resources-count-exporter
+    version: 1.4.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    name: observability-bundle
+    version: 0.8.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd
+    name: k8s-dns-node-cache-app
+    version: 2.4.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: prometheus-blackbox-exporter
+    version: 0.3.2
+  - dependsOn:
+    - prometheus-operator-crd
+    name: cilium-servicemonitors
+    version: 0.1.2
+  - dependsOn:
+    - prometheus-operator-crd
+    name: irsa-servicemonitors
+    version: 0.0.1
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - vertical-pod-autoscaler-crd
+    name: security-bundle
+    version: 0.16.2
```

```suggestion
    version: 0.17.0
```

Do you think we could bump this? 🙏

---

we will include it tomorrow in final cut 👍

---

Thanks, that should be it from our side. Will do another read just in case 🙂